assets/src/application/ui/layout/entityList/pagination/EntityPagination.tsx

```diff
@@ -14,25 +17,29 @@ interface EntityPaginationProps<ELFS extends EntityListFilterStateManager> exten
  * @return EntityPagination
  */
 const EntityPagination: React.FC<EntityPaginationProps<any>> = ({
+	entities,
+	filterState,
 	pageSizeOptions = ['2', '6', '12', '24', '48'],
 	showSizeChanger = true,
 	showTotal,
-	filterState,
 	...rest
 }) => {
-	const { total, setPageNumber, setPerPage, pageNumber }: EntityListFilterStateManager = filterState;
+	const { pageNumber, perPage, setPerPage, setPageNumber, total }: EntityListFilterStateManager = filterState;
+
+	const className = classNames('ee-entity-pagination', { hidePagination: perPage >= entities.length });
```

Comment:
You can simply do this:
```suggestion
	const className = classNames('ee-entity-pagination', { hidePagination: total <= perPage });
```

Reply:
updated, thx